### DISCUSSION TITLE
Problem with Lab 27 Step 3

### DISCUSSION THREAD

**Discussion by charlesmoreno32** [hack4impact-calpoly/starter-pack#6] [Q&A] [answered]
I've run into a little problem with setting up my github repository. I'm getting an error message when I try to type in "git push -u origin main".
<img width="725" alt="Screenshot 2023-09-21 at 5 22 55 PM" src="https://github.com/hack4impact-calpoly/starter-pack/assets/93685786/e186259d-706c-4029-b82f-0fb37e24523c">
I tried figuring out how to use git pull to fix it but nothing has worked so far. I'm thinking it might be caused by the .git at the end of the repository link because my git project doesn't have it nor does my project folder name. 
Edit: I tried using "git remove origin" then redoing it without the .git and it still didn't work.

**Comment by charlesmoreno32** [comment, marked as answer]
I found the answer. The reason is because I clicked in the option of automatically creating a README.md in the github repository. As I remade the repository without it, the branches matched up and everything worked out :)

**Reply by khoa-l** [reply]
Glad, you figured it out :D Thanks for putting the answer here for everyone to see!
